<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTabWidget" name="TabTareas">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tareas</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Categoria</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QComboBox" name="comboCategoria"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Etiqueta</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <widget class="QComboBox" name="comboEtiqueta"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QTableWidget" name="tblTareas">
              <column>
               <property name="text">
                <string>Tarea</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Fecha</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Hecho</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QTextEdit" name="txtTareaDescr">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>200</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="TabCategorias">
         <attribute name="title">
          <string>Categorias</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QTableWidget" name="tblCateg">
              <column>
               <property name="text">
                <string>Categoria</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTextEdit" name="txtCategDescr">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>200</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="TabEtiquetas">
         <attribute name="title">
          <string>Etiquetas</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QTableWidget" name="tblEtiq">
              <column>
               <property name="text">
                <string>Etiqueta</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTareas">
    <property name="title">
     <string>Tareas</string>
    </property>
    <addaction name="actionNuevaTarea"/>
    <addaction name="actionEliminarTarea"/>
   </widget>
   <widget class="QMenu" name="menuCategorias">
    <property name="title">
     <string>Categorias</string>
    </property>
    <addaction name="actionNuevaCateg"/>
    <addaction name="actionEliminarCategoria"/>
   </widget>
   <widget class="QMenu" name="menuEtiquetas">
    <property name="title">
     <string>Etiquetas</string>
    </property>
    <addaction name="actionNuevaEtiq"/>
    <addaction name="actionEliminarEtiqueta"/>
   </widget>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuTareas"/>
   <addaction name="menuCategorias"/>
   <addaction name="menuEtiquetas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNuevaTarea">
   <property name="text">
    <string>Nueva</string>
   </property>
  </action>
  <action name="actionNuevaCateg">
   <property name="text">
    <string>Nueva</string>
   </property>
  </action>
  <action name="actionNuevaEtiq">
   <property name="text">
    <string>Nueva</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionEliminarTarea">
   <property name="text">
    <string>Eliminar</string>
   </property>
  </action>
  <action name="actionEliminarEtiqueta">
   <property name="text">
    <string>Eliminar</string>
   </property>
  </action>
  <action name="actionEliminarCategoria">
   <property name="text">
    <string>Eliminar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionSalir</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
